documentation/assertions/Set/to-have-items-satisfying.md

```diff
@@ -0,0 +1,57 @@
+Asserts the items contained by a Set satisfy a particular list of items.
+
+```js
+expect(new Set([1, 2, 3]), 'to have items satisfying', [1, { foo: 'bar' }, 3]);
```

Comment:
This seems wrong. The `to have items satisfying` family of assertions should be used with an assertion (or literal value) that has to hold true for *every item in the collection*, not the collection as a whole. This looks like an alias for `to [exhaustively] satisfy`?

I think it should be used like this:

```js
expect(new Set([1, 2, 3]), 'to have items satisfying', expect.it('to be a number'));
expect(new Set([1]), 'to have items satisfying', 1);
``

Reply:
Yes, you're right - I misread this. Have adjusted the semantics now :)